__tests__/forces/Gravity_test.re

```diff
@@ -15,9 +15,9 @@ describe("Gravity", () => {
 
     it(
       "should correctly derive the force of gravity at the Earth's surface", () => {
-      let massEarth = 5.97 *. Math.pow(~base=10, ~exp=24);
+      let massEarth = 5.97 *. 10. ** 24.;
```

Comment:
🤦‍♂ OCaml has an exponentation operation already, `**`, so no need to have our own 🎉